packages/react-native-sdk/package.json

```diff
@@ -48,6 +49,9 @@
     "react-native-callkeep": {
       "optional": true
     },
+    "react-native-device-info": {
+      "optional": true
+    },
     "react-native-voip-push-notification": {
```

Comment:
if its an optional dependency we should not throw an error if its not installed

Reply:
Its not an optional dependancy. Its needed for everyone and if anyone doesn't install it we ask them to do it by throwing an error.

Reply:
We have to handle it as optional dependency tho so as to throw an error if not installed. These device info is necessary to be sent from an sdk to the backend and every integrator needs to send it